app/views/partners/show.html.erb

```diff
@@ -34,6 +34,12 @@
             <h2 class="card-title">Partner Actions</h2>
           </div>
           <div class="card-body p-3">
+            <%= link_to partner_users_path(@partner) do %>
+              <div class="btn btn-app bg-success">
+                <i class="fas fa-users"></i> Users
```

Comment:
Please change this to "Manage Users"  to follow the same form for the other actions on the page.